docs/setup-vendor.md

```diff
@@ -0,0 +1,47 @@
+# Vendor's Customers using Private Packagist in a Composer project
+
+Using Private Packagist for Vendors, your customers can install packages with Composer using a unique URL and an authentication token.
+
+Customers can start using your packages in either an existing or a new Composer project.
+
+## Adding a Package to an existing Composer project
+
+To install a package named "acme-company/api" into your existing project, The following steps are required:
```

Comment:
"your" refers to the customer here, but this is supposed to be written for vendors, please review the article again and let me know when you've updated it to be directed at vendors rather than customers.

Reply:
ok, I updated it and made sure the rest of this article is directed at vendors.